.changeset/metal-olives-relax.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-cost-insights': patch
+---
+
+Added optional currencies prop
```

Comment:
```suggestion
Added optional currencies prop for `CostInsightsPage` to allow overriding the default currencies.
```
Maybe just slightly more verbose here, since it will appear alongside other changelogs in the release notes 🙂